src/components/AddTapModal.js

```diff
@@ -0,0 +1,599 @@
+import styles from "./AddTapModal.module.scss";
+import React, { Component } from "react";
+import {
+  Modal,
+  Button,
+  Form,
+  OverlayTrigger,
+  Popover,
+  Accordion
+} from "react-bootstrap";
+import ImageUploader from "react-images-upload";
+import { uploadFile } from "react-s3";
```

Comment:
It looks like the react-s3 library is causing the test to fail on the server. I have not seen any errors locally.

Reply:
Since we switched from the react-s3 library to good 'ol fetch this should no longer be an issue.